src/components/radio/RadioSelect.tsx

```diff
@@ -1,19 +1,49 @@
 import * as React from 'react';
+import * as _ from 'underscore';
 import {ToggleForm} from '../childForm/ToggleForm';
 import {Radio} from './Radio';
 
 export interface IRadioSelectProps {
+    id?: string;
     name?: string;
-    value?: string;
     disabled?: boolean;
-    onChange?: (value: string) => void;
     children?: Array<React.ReactElement<Radio>> | Array<React.ReactElement<ToggleForm>>;
+    /**
+     * used in RadioSelectConnected component only
+     */
+    valueOnMount?: string;
+    disabledValuesOnMount?: string[];
+}
+
+export interface IRadioSelectDispatchProps {
+    onMount?: (id: string, valueOnMount: string, disabledValues: string[]) => void;
+    onUnmount?: (id: string) => void;
+    onChange?: (value: string, id?: string, e?: React.MouseEvent<HTMLElement>) => void;
+}
+
+export interface IRadioSelectStateProps {
+    value?: string;

```

Comment:
Changing this here is kind of a breaking change.

Is there a way to keep `value` in the `IRadioSelectProps` and tag it as deprecated and comment on it like `use IRadioSelectStateProps or IRadioSelectAllProps instead.`?

People that upgrade would get a direction as to what to use instead of just being presented with a compilation error

Reply:
made some changes that now makes it not-breaking